### DISCUSSION TITLE
青春杯从第二战开始无法进入队伍对战。

### DISCUSSION THREAD

**Discussion by punkeek** [NateScarlet/auto-derby#363] [Q&A] [answered]
看起来tap command没有命中按钮捏。
以前1.21版本都没这问题，更新到1.27.5新版本好像不行了.
从青春杯第二战开始就算是重启脚本，也会不命中青春杯按钮。第一战有编辑队伍的弹窗，就100%命中青春杯按钮（分支不同？）。

> Command line message
第一次
![屏幕截图 2022-09-01 020132](https://user-images.githubusercontent.com/50232648/187749104-7b9f9d89-9580-4f7c-a990-9a04d199554c.png)
第二次
![屏幕截图 2022-09-01 021841](https://user-images.githubusercontent.com/50232648/187751256-21f070e0-847a-43d7-8199-2b0e00d93e07.png)

> Game screen（卡在这个界面）
第一次
![屏幕截图 2022-09-01 020156](https://user-images.githubusercontent.com/50232648/187749176-d417b1c9-e233-4e71-ae0a-6844f0a76f04.png)
第二次
![屏幕截图 2022-09-01 021900](https://user-images.githubusercontent.com/50232648/187751283-b9cd2c18-4704-48ad-8dab-71fbcbb188b8.png)

**Comment by punkeek** [comment]
报告有误，从第一战就无法进入队伍对战了

**Comment by NateScarlet** [comment]
不要用图发日志
不要手动截图
[问题反馈](https://github.com/NateScarlet/auto-derby#%E9%97%AE%E9%A2%98%E5%8F%8D%E9%A6%88)

**Comment by punkeek** [comment]
那请问要提供什么呢？

**Reply by NateScarlet** [reply]
链接都给你了

**Comment by punkeek** [comment, marked as answer]
已经转至issue了
